### DISCUSSION TITLE
Out of a sudden, establishing a connection is no longer possible

### DISCUSSION THREAD

**Discussion by Fridolin34** [schwabe/ics-openvpn#1452] [Configuration problems] [answered]
# General information
1. Android Version 10
2. Android Vendor/Custom ROM Samsung
3. Device S9+
4. Version of the app (version number/play store version/self-built) 0.7.33
 
# Description of the issue
On my Synology an OpenVPN server is running and this one is used to receive connection requests from different mobile phones. 
Until yesterday, everything worked quite nicely, but suddenly, my clients aren't even contacting the server anymore.

# Log (if applicable)
2022-02-03 20:25:07 WARNING: No server certificate verification method has been enabled.  See http://openvpn.net/howto.html#mitm for more info.
2022-02-03 20:25:07 OpenSSL: error:0308010C:digital envelope routines::unsupported
2022-02-03 20:25:07 The encryption method of your private keys/pkcs12 might be outdated and you probably need to enable the OpenSSL legacy provider to be able to use this profile.
2022-02-03 20:25:07 MGMT: Got unrecognized command>FATAL:Message hash algorithm 'RSA-SHA512' not found
2022-02-03 20:25:07 MANAGEMENT: Client disconnected
2022-02-03 20:25:07 Message hash algorithm 'RSA-SHA512' not found
2022-02-03 20:25:07 Exiting due to fatal error
2022-02-03 20:25:07 Process exited with exit value 1

OpenSSL legacy provider has been enabled --> nothing changed
In the change log of 0.7.33 it says 'SHA512 has been fixed', but can it be that something has been changed and gotten worse?
I don't know, since when the issue is there, I just tried to establish a connection today and it's not working anymore. Neither the configuration on the mobile device nor on the Synology has been changed - at least not intentionally.

Is there anything I can do? Is anyone having similar issues?

Thanks in advance,
Fridolin

**Comment by schwabe** [comment, marked as answer]
try using SHA512 instead RSA-SHA512. Looks like OpenSSL 3 removed some of the idiotic names.

**Reply by Fridolin34** [reply]
Hi schwabe,

thanks for your answer. Since I'm not an expert regarding network security, could you please tell me what exactly I have to change? In the application I do not see the chance to simply change the name. Do I have to create a new certificate and a new configuration on the server side to update all my clients?

If so, I would say this is pretty weak. How can it be that programmers rely on 'idiotic names' instead of internal identifiers and a simple change of the name causes all clients needing an update? This doesn't sound like professional programming.

Thanks,
Fridolin

**Reply by Fridolin34** [reply]
Hmm...changed the server to 'only' expect SHA512 requests and the configuration in the app accordingly. 

It works -- still I'm pretty disappointed and I don't know, whether this was a breaking change in the app 'OpenVPN for Android' or somewhere else. Furthermore, I've chosen RSA-SHA512 in the first place, because it sounded like a combination or two methods. If you tell me now, that it is the same and only the name has changed, then it all seems less trustworthy.

**Reply by Fridolin34** [reply]
The longer I'm trying to fix that, the more frustrated I'm getting. I have a second Synology set up remotely, having exactly the same configuration. Guess what, I cannot connect to this OpenVPN server with my mobile phone as well - but one of my other Synologys is nicely connected at the same time. 

So the indication just go stronger that the bug is withing the mobile app.

**Reply by schwabe** [reply]
RSA-SHA512 makes no sense as HMAC algorithm from a cryptographic standpoint. It made sense in other part of OpenSSL where RSA and SHA512 are really used. Basically OpenSSL 1.1 accepted RSA-SHA512 in some places where it shouldn't and ignored the RSA part. This has now been fixed in OpenSL 3.0. I never used RSA-SHA* myself so the change was nothing I knew of since using SHA1/SHA384/SHA512 just worked as before.

**Reply by schwabe** [reply]
@Fridolin34 there might be a bug in the app but I would need more details about that one.

**Reply by Fridolin34** [reply]
@schwabe Hi Arne, what else do you need? I've checked the log, but I cannot really identify something useful. 

Either way, I've just added my log lines here:


2022-02-05 18:28:53 official build 0.7.33 running on samsung SM-G965F (exynos9810), Android 10 (QP1A.190711.020) API 29, ABI arm64-v8a, (samsung/star2ltexx/star2lte:10/QP1A.190711.020/G965FXXSHFUJ2:user/release-keys)
2022-02-05 18:28:53 Building configuration…
2022-02-05 18:28:53 started Socket Thread
2022-02-05 18:28:53 Network Status: CONNECTED  to WIFI 
2022-02-05 18:28:53 Debug state info: CONNECTED  to WIFI , pause: userPause, shouldbeconnected: true, network: SHOULDBECONNECTED 
2022-02-05 18:28:53 Debug state info: CONNECTED  to WIFI , pause: userPause, shouldbeconnected: true, network: SHOULDBECONNECTED 
2022-02-05 18:28:53 P:WARNING: linker: Warning: "/data/app/de.blinkt.openvpn-kuwLplaWpNG7RxUq3xoDuQ==/lib/arm64/libovpnexec.so" is not a directory (ignoring)
2022-02-05 18:28:53 WARNING: Compression for receiving enabled. Compression has been used in the past to break encryption. Sent packets are not compressed unless "allow-compression yes" is also set.
2022-02-05 18:28:53 Current Parameter Settings:
2022-02-05 18:28:53   config = '/data/user/0/de.blinkt.openvpn/cache/android.conf'
2022-02-05 18:28:53   mode = 0
2022-02-05 18:28:53   show_ciphers = DISABLED
2022-02-05 18:28:53   show_digests = DISABLED
2022-02-05 18:28:53   show_engines = DISABLED
2022-02-05 18:28:53   genkey = DISABLED
2022-02-05 18:28:53   genkey_filename = '[UNDEF]'
2022-02-05 18:28:53   key_pass_file = '[UNDEF]'
2022-02-05 18:28:53   show_tls_ciphers = DISABLED
2022-02-05 18:28:53   connect_retry_max = 0
2022-02-05 18:28:53 Connection profiles [0]:
2022-02-05 18:28:53   proto = udp
2022-02-05 18:28:53   local = '[UNDEF]'
2022-02-05 18:28:53   local_port = '[UNDEF]'
2022-02-05 18:28:53   remote = 'xxxx.xxxx.xxx'
2022-02-05 18:28:53   remote_port = 'xxxx'
2022-02-05 18:28:53   remote_float = DISABLED
2022-02-05 18:28:53   bind_defined = DISABLED
2022-02-05 18:28:53   bind_local = DISABLED
2022-02-05 18:28:53   bind_ipv6_only = DISABLED
2022-02-05 18:28:53   connect_retry_seconds = 2
2022-02-05 18:28:53   connect_timeout = 120
2022-02-05 18:28:53   socks_proxy_server = '[UNDEF]'
2022-02-05 18:28:53   socks_proxy_port = '[UNDEF]'
2022-02-05 18:28:53   tun_mtu = 1500
2022-02-05 18:28:53   tun_mtu_defined = ENABLED
2022-02-05 18:28:53   link_mtu = 1500
2022-02-05 18:28:53   link_mtu_defined = DISABLED
2022-02-05 18:28:53   tun_mtu_extra = 0
2022-02-05 18:28:53   tun_mtu_extra_defined = DISABLED
2022-02-05 18:28:53   mtu_discover_type = -1
2022-02-05 18:28:53   fragment = 0
2022-02-05 18:28:53   mssfix = 1492
2022-02-05 18:28:53   mssfix_encap = ENABLED
2022-02-05 18:28:53   explicit_exit_notification = 0
2022-02-05 18:28:53   tls_auth_file = '[UNDEF]'
2022-02-05 18:28:53   key_direction = not set
2022-02-05 18:28:53   tls_crypt_file = '[UNDEF]'
2022-02-05 18:28:53   tls_crypt_v2_file = '[UNDEF]'
2022-02-05 18:28:53 Connection profiles END
2022-02-05 18:28:53   remote_random = DISABLED
2022-02-05 18:28:53   ipchange = '[UNDEF]'
2022-02-05 18:28:53   dev = 'tun'
2022-02-05 18:28:53   dev_type = '[UNDEF]'
2022-02-05 18:28:53   dev_node = '[UNDEF]'
2022-02-05 18:28:53   lladdr = '[UNDEF]'
2022-02-05 18:28:53   topology = 1
2022-02-05 18:28:53   ifconfig_local = '[UNDEF]'
2022-02-05 18:28:53   ifconfig_remote_netmask = '[UNDEF]'
2022-02-05 18:28:53   ifconfig_noexec = DISABLED
2022-02-05 18:28:53   ifconfig_nowarn = ENABLED
2022-02-05 18:28:53   ifconfig_ipv6_local = '[UNDEF]'
2022-02-05 18:28:53   ifconfig_ipv6_netbits = 0
2022-02-05 18:28:53   ifconfig_ipv6_remote = '[UNDEF]'
2022-02-05 18:28:53   shaper = 0
2022-02-05 18:28:53   mtu_test = 0
2022-02-05 18:28:53   mlock = DISABLED
2022-02-05 18:28:53   keepalive_ping = 0
2022-02-05 18:28:53   keepalive_timeout = 0
2022-02-05 18:28:53   inactivity_timeout = 0
2022-02-05 18:28:53   ping_send_timeout = 0
2022-02-05 18:28:53   ping_rec_timeout = 0
2022-02-05 18:28:53   ping_rec_timeout_action = 0
2022-02-05 18:28:53   ping_timer_remote = DISABLED
2022-02-05 18:28:53   remap_sigusr1 = 0
2022-02-05 18:28:53   persist_tun = DISABLED
2022-02-05 18:28:53   persist_local_ip = DISABLED
2022-02-05 18:28:53   persist_remote_ip = DISABLED
2022-02-05 18:28:53   persist_key = DISABLED
2022-02-05 18:28:53   passtos = DISABLED
2022-02-05 18:28:53   resolve_retry_seconds = 60
2022-02-05 18:28:53   resolve_in_advance = DISABLED
2022-02-05 18:28:53   username = '[UNDEF]'
2022-02-05 18:28:53   groupname = '[UNDEF]'
2022-02-05 18:28:53   chroot_dir = '[UNDEF]'
2022-02-05 18:28:53   cd_dir = '[UNDEF]'
2022-02-05 18:28:53   writepid = '[UNDEF]'
2022-02-05 18:28:53   up_script = '[UNDEF]'
2022-02-05 18:28:53   down_script = '[UNDEF]'
2022-02-05 18:28:53   down_pre = DISABLED
2022-02-05 18:28:53   up_restart = DISABLED
2022-02-05 18:28:53   up_delay = DISABLED
2022-02-05 18:28:53   daemon = DISABLED
2022-02-05 18:28:53   log = DISABLED
2022-02-05 18:28:53   suppress_timestamps = DISABLED
2022-02-05 18:28:53   machine_readable_output = ENABLED
2022-02-05 18:28:53   nice = 0
2022-02-05 18:28:53   verbosity = 4
2022-02-05 18:28:53   mute = 0
2022-02-05 18:28:53   gremlin = 0
2022-02-05 18:28:53   status_file = '[UNDEF]'
2022-02-05 18:28:53   status_file_version = 1
2022-02-05 18:28:53   status_file_update_freq = 60
2022-02-05 18:28:53   occ = ENABLED
2022-02-05 18:28:53   rcvbuf = 0
2022-02-05 18:28:53   sndbuf = 0
2022-02-05 18:28:53   sockflags = 0
2022-02-05 18:28:53   fast_io = DISABLED
2022-02-05 18:28:53   comp.alg = 2
2022-02-05 18:28:53   comp.flags = 1
2022-02-05 18:28:53   route_script = '[UNDEF]'
2022-02-05 18:28:53   route_default_gateway = '[UNDEF]'
2022-02-05 18:28:53   route_default_metric = 0
2022-02-05 18:28:53   route_noexec = DISABLED
2022-02-05 18:28:53   route_delay = 0
2022-02-05 18:28:53   route_delay_window = 30
2022-02-05 18:28:53   route_delay_defined = DISABLED
2022-02-05 18:28:53   route_nopull = DISABLED
2022-02-05 18:28:53   route_gateway_via_dhcp = DISABLED
2022-02-05 18:28:53   allow_pull_fqdn = DISABLED
2022-02-05 18:28:53   route 0.0.0.0/0.0.0.0/vpn_gateway/default (not set)
2022-02-05 18:28:53   management_addr = '/data/user/0/de.blinkt.openvpn/cache/mgmtsocket'
2022-02-05 18:28:53   management_port = 'unix'
2022-02-05 18:28:53   management_user_pass = '[UNDEF]'
2022-02-05 18:28:53   management_log_history_cache = 250
2022-02-05 18:28:53   management_echo_buffer_size = 100
2022-02-05 18:28:53   management_write_peer_info_file = '[UNDEF]'
2022-02-05 18:28:53   management_client_user = '[UNDEF]'
2022-02-05 18:28:53   management_client_group = '[UNDEF]'
2022-02-05 18:28:53   management_flags = 16678
2022-02-05 18:28:53   shared_secret_file = '[UNDEF]'
2022-02-05 18:28:53   key_direction = not set
2022-02-05 18:28:53   ciphername = 'AES-256-CBC'
2022-02-05 18:28:53   ncp_ciphers = 'AES-256-GCM:AES-128-GCM:AES-256-CBC'
2022-02-05 18:28:53   authname = 'RSA-SHA512'
2022-02-05 18:28:53   engine = DISABLED
2022-02-05 18:28:53   replay = ENABLED
2022-02-05 18:28:53   mute_replay_warnings = DISABLED
2022-02-05 18:28:53   replay_window = 64
2022-02-05 18:28:53   replay_time = 15
2022-02-05 18:28:53   packet_id_file = '[UNDEF]'
2022-02-05 18:28:53   test_crypto = DISABLED
2022-02-05 18:28:53   tls_server = DISABLED
2022-02-05 18:28:53   tls_client = ENABLED
2022-02-05 18:28:53   ca_file = '[INLINE]'
2022-02-05 18:28:53   ca_path = '[UNDEF]'
2022-02-05 18:28:53   dh_file = '[UNDEF]'
2022-02-05 18:28:53   cert_file = '[UNDEF]'
2022-02-05 18:28:53   extra_certs_file = '[UNDEF]'
2022-02-05 18:28:53   priv_key_file = '[UNDEF]'
2022-02-05 18:28:53   pkcs12_file = '[UNDEF]'
2022-02-05 18:28:53   cipher_list = '[UNDEF]'
2022-02-05 18:28:53   cipher_list_tls13 = '[UNDEF]'
2022-02-05 18:28:53   tls_cert_profile = 'legacy'
2022-02-05 18:28:53   tls_verify = '[UNDEF]'
2022-02-05 18:28:53 Waiting 0s seconds between connection attempt
2022-02-05 18:28:53   tls_export_cert = '[UNDEF]'
2022-02-05 18:28:53   verify_x509_type = 0
2022-02-05 18:28:53   verify_x509_name = '[UNDEF]'
2022-02-05 18:28:53   crl_file = '[UNDEF]'
2022-02-05 18:28:53   ns_cert_type = 0
2022-02-05 18:28:53   remote_cert_ku[i] = 0
2022-02-05 18:28:53   remote_cert_ku[i] = 0
2022-02-05 18:28:53   remote_cert_ku[i] = 0
2022-02-05 18:28:53   remote_cert_ku[i] = 0
2022-02-05 18:28:53   remote_cert_ku[i] = 0
2022-02-05 18:28:53   remote_cert_ku[i] = 0
2022-02-05 18:28:53   remote_cert_ku[i] = 0
2022-02-05 18:28:53   remote_cert_ku[i] = 0
2022-02-05 18:28:53   remote_cert_ku[i] = 0
2022-02-05 18:28:53   remote_cert_ku[i] = 0
2022-02-05 18:28:53   remote_cert_ku[i] = 0
2022-02-05 18:28:53   remote_cert_ku[i] = 0
2022-02-05 18:28:53   remote_cert_ku[i] = 0
2022-02-05 18:28:53   remote_cert_ku[i] = 0
2022-02-05 18:28:53   remote_cert_ku[i] = 0
2022-02-05 18:28:53   remote_cert_ku[i] = 0
2022-02-05 18:28:53   remote_cert_eku = '[UNDEF]'
2022-02-05 18:28:53   ssl_flags = 192
2022-02-05 18:28:53   tls_timeout = 2
2022-02-05 18:28:53   renegotiate_bytes = -1
2022-02-05 18:28:53   renegotiate_packets = 0
2022-02-05 18:28:53   renegotiate_seconds = 0
2022-02-05 18:28:53   handshake_window = 60
2022-02-05 18:28:53   transition_window = 3600
2022-02-05 18:28:53   single_session = DISABLED
2022-02-05 18:28:53   push_peer_info = DISABLED
2022-02-05 18:28:53   tls_exit = DISABLED
2022-02-05 18:28:53   tls_crypt_v2_metadata = '[UNDEF]'
2022-02-05 18:28:53   server_network = 0.0.0.0
2022-02-05 18:28:53   server_netmask = 0.0.0.0
2022-02-05 18:28:53   server_network_ipv6 = ::
2022-02-05 18:28:53   server_netbits_ipv6 = 0
2022-02-05 18:28:53   server_bridge_ip = 0.0.0.0
2022-02-05 18:28:53   server_bridge_netmask = 0.0.0.0
2022-02-05 18:28:53   server_bridge_pool_start = 0.0.0.0
2022-02-05 18:28:53   server_bridge_pool_end = 0.0.0.0
2022-02-05 18:28:53   ifconfig_pool_defined = DISABLED
2022-02-05 18:28:53   ifconfig_pool_start = 0.0.0.0
2022-02-05 18:28:53   ifconfig_pool_end = 0.0.0.0
2022-02-05 18:28:53   ifconfig_pool_netmask = 0.0.0.0
2022-02-05 18:28:53   ifconfig_pool_persist_filename = '[UNDEF]'
2022-02-05 18:28:53   ifconfig_pool_persist_refresh_freq = 600
2022-02-05 18:28:53   ifconfig_ipv6_pool_defined = DISABLED
2022-02-05 18:28:53   ifconfig_ipv6_pool_base = ::
2022-02-05 18:28:53   ifconfig_ipv6_pool_netbits = 0
2022-02-05 18:28:53   n_bcast_buf = 256
2022-02-05 18:28:53   tcp_queue_limit = 64
2022-02-05 18:28:53   real_hash_size = 256
2022-02-05 18:28:53   virtual_hash_size = 256
2022-02-05 18:28:53   client_connect_script = '[UNDEF]'
2022-02-05 18:28:53   learn_address_script = '[UNDEF]'
2022-02-05 18:28:53   client_disconnect_script = '[UNDEF]'
2022-02-05 18:28:53   client_config_dir = '[UNDEF]'
2022-02-05 18:28:53   ccd_exclusive = DISABLED
2022-02-05 18:28:53   tmp_dir = '/data/data/de.blinkt.openvpn/cache'
2022-02-05 18:28:53   push_ifconfig_defined = DISABLED
2022-02-05 18:28:53   push_ifconfig_local = 0.0.0.0
2022-02-05 18:28:53   push_ifconfig_remote_netmask = 0.0.0.0
2022-02-05 18:28:53   push_ifconfig_ipv6_defined = DISABLED
2022-02-05 18:28:53   push_ifconfig_ipv6_local = ::/0
2022-02-05 18:28:53   push_ifconfig_ipv6_remote = ::
2022-02-05 18:28:53   enable_c2c = DISABLED
2022-02-05 18:28:53   duplicate_cn = DISABLED
2022-02-05 18:28:53   cf_max = 0
2022-02-05 18:28:53   cf_per = 0
2022-02-05 18:28:53   max_clients = 1024
2022-02-05 18:28:53   max_routes_per_client = 256
2022-02-05 18:28:53   auth_user_pass_verify_script = '[UNDEF]'
2022-02-05 18:28:53   auth_user_pass_verify_script_via_file = DISABLED
2022-02-05 18:28:53   auth_token_generate = DISABLED
2022-02-05 18:28:53   auth_token_lifetime = 0
2022-02-05 18:28:53   auth_token_secret_file = '[UNDEF]'
2022-02-05 18:28:53   port_share_host = '[UNDEF]'
2022-02-05 18:28:53   port_share_port = '[UNDEF]'
2022-02-05 18:28:53   vlan_tagging = DISABLED
2022-02-05 18:28:53   vlan_accept = all
2022-02-05 18:28:53   vlan_pvid = 1
2022-02-05 18:28:53   client = ENABLED
2022-02-05 18:28:53   pull = ENABLED
2022-02-05 18:28:53   auth_user_pass_file = 'stdin'
2022-02-05 18:28:53 OpenVPN 2.6-icsopenvpn [git:icsopenvpn/v0.7.33-0-g8bc2287a] arm64-v8a [SSL (OpenSSL)] [LZO] [LZ4] [EPOLL] [MH/PKTINFO] [AEAD] built on Jan 13 2022
2022-02-05 18:28:53 library versions: OpenSSL 3.0.1 14 Dec 2021, LZO 2.10
2022-02-05 18:28:53 MANAGEMENT: Connected to management server at /data/user/0/de.blinkt.openvpn/cache/mgmtsocket
2022-02-05 18:28:53 MANAGEMENT: CMD 'version 3'
2022-02-05 18:28:53 MANAGEMENT: CMD 'hold release'
2022-02-05 18:28:53 MANAGEMENT: CMD 'username 'Auth' XX'
2022-02-05 18:28:53 MANAGEMENT: CMD 'bytecount 2'
2022-02-05 18:28:53 MANAGEMENT: CMD 'state on'
2022-02-05 18:28:53 MANAGEMENT: CMD 'password [...]'
2022-02-05 18:28:53 MANAGEMENT: CMD 'proxy NONE'
2022-02-05 18:28:54 WARNING: No server certificate verification method has been enabled.  See http://openvpn.net/howto.html#mitm for more info.
2022-02-05 18:28:54 OpenSSL: error:0308010C:digital envelope routines::unsupported
2022-02-05 18:28:54 The encryption method of your private keys/pkcs12 might be outdated and you probably need to enable the OpenSSL legacy provider to be able to use this profile.
2022-02-05 18:28:54 MGMT: Got unrecognized command>FATAL:Message hash algorithm 'RSA-SHA512' not found
2022-02-05 18:28:54 MANAGEMENT: Client disconnected
2022-02-05 18:28:54 Message hash algorithm 'RSA-SHA512' not found
2022-02-05 18:28:54 Exiting due to fatal error
2022-02-05 18:28:54 Process exited with exit value 1

**Reply by schwabe** [reply]
It is the RSA-SHA512 vs SHA512. Just replace the RSA-SHA512 with SHA512 in your config.

**Reply by Fridolin34** [reply]
Sorry, not getting it. Yes, I've changed the configuration of the server and the client and then it worked. So it is not only cutting of the RSA- in the client config and it would work. The server has to be changed as well.

But I have two VPN servers running and one of them is remotely placed. So it is slightly inconvenient for me to change that, because I would need to go there and change it, just to be able to connect to this server again.

Basically, the server does not need to be changed, because another server is still perfectly connected to this remote VPN server.

Therefore, why has the mobile application been changed and when is this going to be fixed? 
What kind of additional information you've requested two posts ago could help you?

**Reply by schwabe** [reply]
Check also this: https://security.stackexchange.com/questions/91908/using-rsa-sha-as-instead-hmac-in-openvpn

The OpenSSL version has been updated. The change was an update. And please complain on the OpenSSL upstream if you think that removing RSA-SHA512 alias is a bug. Just changing the client side should work though and you haven't posted a log that would suggest otherweise.

**Comment by Deepanshusuman** [comment]
same i am using aes-256-cbc

**Reply by schwabe** [reply]
@Deepanshusuman as --auth method?

**Comment by Deepanshusuman** [comment]
```
client
dev tun
proto tcp
sndbuf 0
rcvbuf 0
remote server port
resolv-retry infinite
nobind
persist-key
persist-tun
remote-cert-tls server
auth SHA512
cipher AES-256-CBC
setenv opt block-outside-dns
key-direction 1
verb 3
<ca>
-----BEGIN CERTIFICATE-----
certificate
-----END CERTIFICATE-----
</ca>
<cert>
-----BEGIN CERTIFICATE-----
certificate
-----END CERTIFICATE-----
</cert>
<key>
-----BEGIN PRIVATE KEY-----
key
-----END PRIVATE KEY-----
</key>
<tls-auth>
-----BEGIN OpenVPN Static key V1-----
key
-----END OpenVPN Static key V1-----
</tls-auth>
```

**Comment by schwabe** [comment]
@Deepanshusuman can you post the log from the client?

**Comment by Deepanshusuman** [comment]
Could not identify launch activity: Default Activity not found
Error while Launching activity
Failed to launch an application on all devices

**Reply by schwabe** [reply]
you get that message when you click on the app icon in the launcher? You need to provide more details as I am not sure what is happening on your side as the "same" indicating you get the same error as the OP and now there is a compeltely different error mesage.

**Comment by Deepanshusuman** [comment]


**Reply by schwabe** [reply]
That is a sucessful connection.

**Reply by Deepanshusuman** [reply]
yes my bad it is a server side issue
